src/components/constants/SampleText.js

```diff
@@ -0,0 +1,12 @@
+const sampleText = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum
+  aliquet felis ut sem ornare, quis facilisis diam egestas. Integer
+  venenatis cursus nisi. Nulla feugiat ligula et ligula fringilla viverra.
+  In purus mauris, congue sodales tincidunt non, accumsan in est. In orci
+  turpis, scelerisque aliquet diam ac, vehicula mattis enim. Praesent
+  congue quam nunc, eu sollicitudin ipsum elementum ac. Sed semper massa
+  at purus consectetur sollicitudin. Aliquam id luctus tortor. Integer sit
+  amet elementum dolor, in pharetra diam. Donec cursus justo nec placerat
+  egestas. Sed fringilla est a diam porttitor ullamcorper. Cras vulputate
+  tincidunt sem ut vehicula.`;
+
+export default sampleText;
```

Comment:
No newline at EOF.

Reply:
Thanks, @lycoangelo. This is updated now.